locales/fr.json

```diff
@@ -185,31 +185,31 @@
-  "Graph will not include events": "Graph will not include events",
-  "ENS domain to associate to": "ENS domain to associate to",
-  "output usage information and help information": "output usage information and help information",
-  "Starting %s process": "Starting %s process",
-  "Starting Blockchain node in another process": "Starting Blockchain node in another process",
-  "Blockchain node is ready": "Blockchain node is ready",
-  "Storage process for {{storageName}} ended before the end of this process. Code: {{code}}": "Storage process for {{storageName}} ended before the end of this process. Code: {{code}}",
-  "ipfs process started": "ipfs process started",
-  "Compiling...": "Compiling...",
-  "Building...": "Building...",
-  "gas at the price of": "gas at the price of",
-  "Wei, estimated cost:": "Wei, estimated cost:",
-  "using": "using",
-  "Embark is building, please wait...": "Embark is building, please wait...",
-  "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)": "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)",
-  "Initializing Embark Template...": "Initializing Embark Template...",
-  "Name your app (default is %s):": "Name your app (default is %s):",
-  "Unrecognized balance string \"%s\"": "Unrecognized balance string \"%s\"",
-  "{{className}} has a dependency on {{dependencyName}}": "{{className}} has a dependency on {{dependencyName}}",
-  ", but it is not set to deploy. It could be an interface.": ", but it is not set to deploy. It could be an interface.",
-  "Done. %s generated": "Done. %s generated"
+  "language to use (default: en)": "langue à utiliser (langue principale: en)",
+  "download template": "modèle de téléchargement",
+  "Start the Embark console": "Démarrer la console Embark",
+  "When set, will print the gas cost for each contract deploy": "Une fois défini, imprimera le coût du gaz pour chaque déploiement de contrat",
+  "Graph will not include undeployed contracts": "Le graphique n'inclut pas les contrats non déployés",
+  "Graph will not include functions": "Le graphique n'inclut pas les fonctions",
+  "Graph will not include events": "Le graphique n'inclut pas les événements",
+  "ENS domain to associate to": "Domaine ENS à associer à",
+  "output usage information and help information": "informations sur l'utilisation de la sortie et aide",
+  "Starting %s process": "Début du processus %s",
+  "Starting Blockchain node in another process": "Démarrage du nœud Blockchain dans un autre processus",
+  "Blockchain node is ready": "Le nœud de la blockchain est prêt",
+  "Storage process for {{storageName}} ended before the end of this process. Code: {{code}}": "Le processus de stockage pour {{storageName}} a pris fin avant la fin de ce processus. Code: {{code}}",
+  "ipfs process started": "processus ipfs commencé",
+  "Compiling...": "Compiler...",
+  "Building...": "Bâtiment...",
```

Comment:
@schweigert "Bâtiment" means "Edificio" /  "Predio" in portuguese.

Reply:
@iurimatias Would "construire" be the correct word?